deploy/packer/base/scripts/packages.sh

```diff
@@ -13,17 +13,38 @@ sudo usermod -a -G docker $USER
   apt-get install apt-transport-https
 }
 
+# For reasons that aren't at all clear, packer obliterates sources.list for paravirtualized AMIs.
+# So we put it back with apt-get clean
+sudo apt-get clean
+sudo apt-get update -y
```

Comment:
There is also the possibility of apt-get update running on line 12.  Could this block just be moved up before line 11 and then remove line 12?